web3_proxy/src/frontend/authorization.rs

```diff
@@ -1083,6 +1093,34 @@ impl Web3ProxyApp {
         }
     }
 
+    /// Get the balance for the user.
+    ///
+    /// If a subuser calls this function, the subuser needs to have first attained the user_id that the rpc key belongs to.
+    /// This function should be called anywhere where balance is required (i.e. only rpc calls, I believe ... non-rpc calls don't really require balance)
+    pub(crate) async fn balance_checks(&self, user_id: u64) -> Web3ProxyResult<Decimal> {
+        // Downgrade the user if the balance is too low ...
```

Comment:
Is this comment accurate? I don't see any downgrade logic here. It's just getting the balance, right?

Reply:
no it's not, i removed it